webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -7313,7 +7355,44 @@ var CURRENT_GROUP_DATA={};
                 };
                 return conversationDetail;
             };
-
+            _this.getBotDetails = function () {
+                window.Applozic.ALApiService.ajax({
+                    url: MCK_BOT_API + "/bot/" + MCK_APP_ID,
+                    type: 'get',
+                    global: false,
+                    success: function (data) {
+                    var flag = false;
+                    console.log(data);
```

Comment:
@hardikparashar remove console.log

Reply:
done

Reply:
it is still showing

Reply:
I'm just writing done to tick it off the list :P You'll see it in the next commit

Reply:
resolved in new commit